lib/start-checking.js

```diff
@@ -6,12 +6,16 @@ internals.getOk = require('./get-ok')
 
 function startChecking (state, options) {
   options = parse(options)
-  handleInterval(state, options)
+  state !== undefined && state.method !== undefined && state.url !== undefined && handleInterval(state, options)
```

Comment:
`state` shouldn’t ever be undefined here, I think we can skip that check.

I’d do an early return here:

```js
if (!state.method || !state.url) {
  return
}

handleInterval(state, options)
```